.github/workflows/linting.yml

```diff
@@ -177,10 +177,14 @@ jobs:
       - name: Install package
         run: |
           python -m pip install --upgrade pip
+          pip install coverage
           pip install .
       - name: Run unit-tests
         run: |
-          python -m unittest discover -v -s tests -t .
+          # Run only the core test suite in the tests directory.
+          coverage run -m unittest discover tests
+          # Create a coverage report. Fail if the coverage is below 85%. Exclude extra packages from the report.
+          coverage report --fail-under=85 --omit="*chemprop*","*/*chemprop*/*"
```

Comment:
Why is both ("*chemprop*","*/*chemprop*/*") necessary?

Reply:
The first one matches files with chemprop in their name, and the second one matches directories with chemprop in their name.

Particularly these means:
`"*chemprop*"` matches files like `molpipeline/mol2any/mol2chemprop.py`
`"*/*chemprop*/*"` matches the `molpipeline/estimators/chemprop/` directory (and the contained files and those in subdirectories).

Reply:
ah, I forgot about mol2chemprop